templates/static/js/src/demobox/components/LiveDemoBox/LiveDemoBox.tsx

```diff
@@ -18,44 +18,121 @@ type InitialState = {
   switch: [InputOptions];
 };
 
+type SelectValueType = {
+  [key: string]: string | boolean;
+};
+
 const LiveDemoBox = () => {
   const [configValues, setConfigValues] = useState<InitialState>();
+  const [selectValue, setSelectValues] = useState<SelectValueType>({
+    type: "",
+    theme: "",
+    style: "",
```

Comment:
A minor detail, but I think that `dismiss`, `actions` and `timestamp` should be included here as well.

Reply:
We weren't sure how to do this, as the values of these are going to be different depending on the variant. type theme and style will always be consistent, but the other values will be different

Reply:
That makes sense, it should be fine to leave it as it is